File: market-application/src/main/java/com/example/market/application/port/in/RecordInspectionResultUseCase.java
package com.example.market.application.port.in;

import com.example.market.application.command.RecordInspectionResultCommand;
import com.example.market.domain.inspection.InspectionRequest;

public interface RecordInspectionResultUseCase {
    InspectionRequest record(RecordInspectionResultCommand command);
}

File: market-application/src/main/kotlin/com/example/market/application/port/in/RecordInspectionResultUseCase.kt
package com.example.market.application.port.`in`

import com.example.market.application.command.RecordInspectionResultCommand
import com.example.market.domain.inspection.InspectionRequest

interface RecordInspectionResultUseCase {
    fun record(command: RecordInspectionResultCommand): InspectionRequest
}
